dms_field/static/src/xml/preview.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-name="dms.DocumentTreeViewDirectoryPreview">
+        <div class="o_preview_directory">
+            <div class="o_preview_directory_header">
+                <h3>
+                    <t t-esc="dms_object.data.name" />
+                </h3>
+            </div>
+            <div class="o_preview_directory_body row">
+                <div class="col-sm-3 o_preview_directory_icon row" align="center">
```

Comment:
With `col-sm-5` here and `col-sm-7` in line 36, everything fits better

Reply:
I think the current solution is even clearer :wink: